public/stylesheets/footer.css

```diff
@@ -0,0 +1,11 @@
+.myFooter {
+  width: 100%;
+  display: flex;
+  overflow: hidden;
```

Comment:
This seems unnecessary. If the content is making the footer overflow, consider making changes to the content

Reply:
Guess it really was too late into the night when i was doing this footer part, remember ever writing this, will fix it though.